deploy/pipeline/docs/tutorials/ppvehicle_attribute_en.md

```diff
@@ -0,0 +1,121 @@
+English | [简体中文](ppvehicle_attribute.md)
+
+# Attribute Recognition Module of PP-Vehicle
+
+Vehicle attribute recognition is widely used in smart cities, smart transportation and other scenarios. In PP-Vehicle, a vehicle attribute recognition module is integrated, which can identify vehicle color and model.
```

Comment:
PP-Vehicle integrates a vehicle attribute recognition module to identify vehicle color and models.